README.md

```diff
@@ -62,12 +62,16 @@ Create a monster
 
 ```
 POST http://localhost:3000/monsters
+
 headers: 
 {
   "Content-Type": "application/json",
   Accept: "application/json"
 }
 
-data:
-{ name: string, age: number, description: string }
+body: JSON.stringify({ 
+  name: string, 
```

Comment:
try adding <> between each of the parameters so the students know to replace them.
for example:
name: <string>

Reply:
updated